src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    public String getRandomColor(int numberColor) {
+        String[] colors = new String[]{"red", "grey", "white", "black", "yellow", "orange", "blue"};
+        return colors[numberColor];
```

Comment:
```suggestion
        int randomColorIndex = new Random().nextInt(colors.length - 1);
        return colors[randomColorIndex];
```
так можна спростити і не доведеться передавати ззовні інт. Бо з поточноою імплементацією якщо хтось захоче заюзати цей клас завжди треба дивитись, що всередині щоб не зловити out of bound